### PR TITLE
add javadoc stuff

### PR DESCRIPTION
**Pull request by powercasgamer**
lmk if there's a better way for now

### CODE REVIEW

**Comment by olijeffers0n** [comment]
lol, gotta put the code ticks inside the square brackets

**Comment by olijeffers0n** [comment]
Why is this a draft?

**Comment by Leguan16** [comment]
because its probably not everything?

**Comment by olijeffers0n** [comment]
These look like a good improvement, can I get this ready for review

File: docs/paper/dev/api/folia-support.mdx [hunk 1]
```diff
@@ -39,15 +39,15 @@ that can be used for different things. They are:
 If you use these schedulers when running Paper, they will be internally handled to provide the same functionality as if you were
 running Folia.
 
-### Global Scheduler
+### <VersionedJavaDocLink target={"/io/papermc/paper/threadedregions/scheduler/GlobalScheduler.html"}>Global Scheduler</VersionedJavaDocLink>
```
Comment (Nacioszeczek) [inline docs/paper/dev/api/folia-support.mdx:42]: Making titles links feels wrong

File: docs/paper/dev/api/folia-support.mdx [hunk 2]
```diff
@@ -39,15 +39,15 @@ that can be used for different things. They are:
...
 The tasks that you run on the Global Scheduler will be executed on the global region, see [here](/folia/reference/overview#global-region) for
 more information. You should use this scheduler for any tasks that do not belong to any particular region. These can be fetched with:
 ```java
 GlobalRegionScheduler globalScheduler = server.getGlobalRegionScheduler();
 ```
 
-### Region Scheduler
-The region scheduler will be in charge of running tasks for the region that owns a certain location. Do not use this scheduler for 
+### <VersionedJavaDocLink target={"/io/papermc/paper/threadedregions/scheduler/RegionScheduler.html"}>Region Scheduler</VersionedJavaDocLink>
```
Comment (Nacioszeczek) [inline docs/paper/dev/api/folia-support.mdx:49]: Same here

File: docs/paper/dev/api/folia-support.mdx [hunk 3]
```diff
@@ -59,15 +59,15 @@ scheduler.execute(plugin, locationToChange, () -> {
 });
 ```
 
-We pass the location as a parameter to the `RegionScheduler` as it needs to work out which region to execute on.
+We pass the location as a parameter to the <VersionedJavaDocLink target={"/io/papermc/paper/threadedregions/scheduler/RegionScheduler.html"}>RegionScheduler</VersionedJavaDocLink> as it needs to work out which region to execute on.
 
-### Async Scheduler
+### <VersionedJavaDocLink target={"/io/papermc/paper/threadedregions/scheduler/AsyncScheduler.html"}>Async Scheduler</VersionedJavaDocLink>
```
Comment (Nacioszeczek) [inline docs/paper/dev/api/folia-support.mdx:64]: Same here

File: docs/paper/dev/api/folia-support.mdx [hunk 4]
```diff
@@ -59,15 +59,15 @@ scheduler.execute(plugin, locationToChange, () -> {
...
 The Async Scheduler can be used for running tasks independent of the server tick process. This can be fetched with:
 ```java
 AsyncScheduler asyncScheduler = server.getAsyncScheduler();
 ```
 
-### Entity Scheduler
+### <VersionedJavaDocLink target={"/io/papermc/paper/threadedregions/scheduler/EntityScheduler.html"}>Entity Scheduler</VersionedJavaDocLink>
```
Comment (Nacioszeczek) [inline docs/paper/dev/api/folia-support.mdx:70]: Same here

File: docs/velocity/admin/getting-started/README.md [hunk 5]
```diff
@@ -9,7 +9,7 @@ This page covers how to install and set up a minimal configuration of Velocity.
 ## Installing Java
 
 Velocity is written in Java, so if you do not already have Java installed, you will need to install
-it before you continue. Velocity requires Java 11 or newer. See our
+it before you continue. Velocity requires Java 17 or newer. See our
```
Comment (Nacioszeczek) [inline docs/velocity/admin/getting-started/README.md:12]: Doesn't exactly fit into the title 'add javadoc stuff'

File: docs/velocity/dev/api/scheduler.mdx [hunk 6]
```diff
@@ -31,7 +30,7 @@ Time arguments are specified as a `long` with a `java.util.concurrent.TimeUnit`.
 makes scheduling delayed tasks more readable and allows for greater precision.
 `2L, TimeUnit.SECONDS` is far easier to understand than the ambiguous `2000L`.
 
-You can also use a `java.time.Duration` to specify the time arguments, e.g.: `Duration.ofSeconds(5L)`.
+You can also use a [java.time.Duration](https://docs.oracle.com/en/java/javase/21/docs/api/java.base/java/time/Duration.html) to specify the time arguments, e.g.: `Duration.ofSeconds(5L)`.
```
Comment (Nacioszeczek) [inline docs/velocity/dev/api/scheduler.mdx:33]: We can yeet the `java.time.` if we're linking to it

**Review by Nacioszeczek — CHANGES_REQUESTED** [review]

File: docs/paper/dev/api/folia-support.mdx (hunk 1 above)
Comment (olijeffers0n) [reply docs/paper/dev/api/folia-support.mdx:42]: On the line below the Scheduler name also exists so placing it there would work

**Pull request closed without merging**